### DISCUSSION TITLE
Can't start using, always getting "dial tcp 127.0.0.1:27015: connectex: No connection could be made"

### DISCUSSION THREAD

**Discussion by beseg** [danielpaulus/go-ios#145] [General]
Hi, I want to start using go-ios. I use Windows 10.
I installed node-js&npm (https://phoenixnap.com/kb/install-node-js-npm-on-windows). Then I ran `npm install -g go-ios`. Now I can run `ios --version` and get `{"version":"v1.0.70"}`, but no other command can be run. 

For instance:

ios list - {"err":"dial tcp 127.0.0.1:27015: connectex: No connection could be made because the target machine actively refused it.","level":"fatal","msg":"failed getting device list","time":"..."}

ios info - {"err":"dial tcp 127.0.0.1:27015: connectex: No connection could be made because the target machine actively refused it.","level":"fatal","msg":"error getting devicelist","time":"..."}

ios listen - {"level":"error","msg":"could not connect to /var/run/usbmuxd with err dial tcp 127.0.0.1:27015: connectex: No connection could be made because the target machine actively refused it., will retry in 3 seconds...","time":"..."}

What am I missing?

**Comment by danielpaulus** [comment]
You need to install itunes. It comes with a USB driver that allows connecting to iOS devices on TCP port 27015
